### ISSUE TITLE
Can I expect high performance on classification tasks while using cos_sim as score function in training stage?

### ISSUE THREAD

**Issue by LeeJiangWei** [UKPLab/sentence-transformers#1231]
Hi, I want to leverage MultipleNegativeRankingLoss for my classification datasets. Can I train a model with  MultipleNegativeRankingLoss then finetune it with softmax loss? Or should I pass the linear classifer as scoring function (rather than cos_sim) in MultipleNegativeRankingLoss?
